js/private/js_run_devserver.bzl

```diff
@@ -109,6 +115,7 @@ def js_run_devserver(
         grant_sandbox_write_permissions = False,
         use_execroot_entry_point = True,
         allow_execroot_entry_point_with_no_copy_data_to_bin = False,
+        js_run_devserver_rule = _js_run_devserver,
```

Comment:
Wasn't sure of a better way to do this, but rather than duplicating the `js_run_devserver` macro in `js/private/test/js_run_devserver/js_run_devserver_test.bzl`, I added an attribute to accept the rule that should be used (which of course defaults to the `_js_run_devserver` rule)

Let me know if there's a better way to achieve something like this though.